common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -34,6 +37,9 @@ def setUp(self, is_staff=True):
         self.content_group_a = "Content Group A" + XSS_INJECTION
         self.content_group_b = "Content Group B" + XSS_INJECTION
 
+        # Creates the Course modes needed to test enrollment tracks
+        add_enrollment_course_modes(self, [AUDIT_TRACK, VERIFIED_TRACK])
```

Comment:
As I mentioned on the helper method, I think these should just be ["audit", "verified"] (slugs).